spec.html

```diff
@@ -32,4 +32,106 @@
   }
 </style>
 
-TODO
+<emu-clause id="sec-native-error-types-used-in-this-standard-aggregateerror">
+  <h1>AggregateError ( errors, message )</h1>
+  <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
+  <emu-alg>
+    1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+    1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateErrorPrototype%"`, « [[ErrorData]] »).
```

Comment:
@ljharb should it be
```suggestion
    1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
```
?
ref: https://github.com/tc39/ecma262/pull/1376/files

Reply:
Oh, good one

Reply:
@chicoxyzzy once that PR lands, yes, it decidedly should be :-) until then, tho, you'd need to create a new intrinsic for the prototype for this spec to be correct.